tests/target/issue-4452.rs

```diff
@@ -0,0 +1,15 @@
+fn a(&self) -> i64 {
+    #[allow(clippy::cast_possible_wrap)]
```

Comment:
Remember that we need to include comments on these
```suggestion
    // foo
    #[allow(clippy::cast_possible_wrap)]
```